pcsx2/MTGS.cpp

```diff
@@ -202,8 +202,7 @@ void SysMtgsThread::OpenPlugin()
 	else
 		result = GSopen( (void*)pDsp, "PCSX2", renderswitch ? 2 : 1 );
 
-
-	GSsetVsync(EmuConfig.GS.GetVsync());
+	GSsetVsync(EmuConfig.GS.FrameLimitEnable ? EmuConfig.GS.GetVsync() : 0);
```

Comment:
I would prefer if the ``FrameLimitEnable`` check is done inside ``EmuConfig.GS.GetVsync()``. (Inside the function, we're already returning 0 for turbo mode, doesn't hurt to also extend it to framelimit)